### DISCUSSION TITLE
@rollup/plugin-swc does not allow input files to be filtered

### DISCUSSION THREAD

**Discussion by BryceMehring** [rollup/plugins#1565] [Q&A]
- Rollup Plugin Name: @rollup/plugin-swc
- Rollup Plugin Version: 0.1.1
- Rollup Version: 3.28.0
- Operating System (or Browser): MacOS
- Node Version: 16.20.2

### Expected Behavior
Allow input files to be filtered for @rollup/plugin-swc. I am building a web app and want to exclude files from being process by SWC like CSS imports for example. This plugin should function like other `@rollup` plugins by using `createFilter` from `@rollup/pluginutils` and filtering based on the `id` in the `transform` plugin function.

### Actual Behavior

All files get process by SWC. See https://github.com/rollup/plugins/blob/swc-v0.1.1/packages/swc/src/module.ts#L27

**Comment by tada5hi** [comment]
#1566
